app/src/main/java/com/example/miniproject/data/source/ImageRepository.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.miniproject.data.source
+
+import android.os.AsyncTask
+import com.example.miniproject.data.model.ResponseImage
+
+class ImageRepository( private var remoteDataSource: ImageDataSource): ImageDataSource {
```

Comment:
Singleton pattern cần private constructor 
```suggestion
class ImageRepository private constructor( 
    private val remoteDataSource: ImageDataSource
): ImageDataSource {
```